Sofa/GUI/Qt/src/sofa/gui/qt/RealGUI.cpp

```diff
@@ -1424,12 +1403,10 @@ void RealGUI::initViewer(BaseViewer* _viewer)
     sofa::gui::qt::viewer::SofaViewer* sofaViewer = dynamic_cast<sofa::gui::qt::viewer::SofaViewer*>(_viewer);
     if( sofaViewer == nullptr )
     {
-        isEmbeddedViewer(false);
         std::cout<<"initViewer: The viewer isn't embedded in the GUI"<<std::endl;
```

Comment:
As embedding is more or less mandatory in your PR, you should change the message, as it is not relevant anymore.
I would even set an error, as I think it was for case where people wanted none-Qt  (i.e non-QOBject) viewer to be used with a Qt Gui (?)

Reply:
no idea ^^
just some old clean